crud/select.lua

```diff
@@ -22,29 +23,36 @@ local select_module = {}
 
 local SELECT_FUNC_NAME = '__select'
 
-local function call_select_on_storage(space_name, conditions, opts)
-    checks('string', '?table', {
-        limit = '?number',
+local function call_select_on_storage(space_name, index_id, scan_value, iter, conditions, opts)
+    checks('string', 'number', '?table', 'number', 'table', {
+        scan_condition_num = '?number',
         after_tuple = '?table',
+        batch_size = 'number',
```

Comment:
I believe it should be limit.
Also `batch_size = 1ULL` will raise because `1ULL/LL` is cdata

Reply:
Fixed in the commit below.